<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinderDialog</class>
 <widget class="QDialog" name="FinderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QToolButton" name="tbDrives">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSubDirs">
         <property name="text">
          <string>Sub Dirs</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Search in:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePath"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="searchforlayout">
         <item>
          <widget class="QLabel" name="lPattern">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Search for:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePattern"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gbMethods">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Methods</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="cbSha1">
            <property name="text">
             <string>Sha1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbSize">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbName">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
      <widget class="QCheckBox" name="cbDuplicatesEnbl">
       <property name="text">
        <string>Duplicates</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbSelectAll">
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbUnselectAll">
        <property name="text">
         <string>Unsellect All</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbDeleteSelected">
        <property name="text">
         <string>Delete Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Selected files size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lSFS">
        <property name="text">
         <string>0 B</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="twFileList">
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>16</number>
     </attribute>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lePath</tabstop>
  <tabstop>tbDrives</tabstop>
  <tabstop>cbSubDirs</tabstop>
  <tabstop>lePattern</tabstop>
  <tabstop>cbDuplicatesEnbl</tabstop>
  <tabstop>cbSha1</tabstop>
  <tabstop>cbSize</tabstop>
  <tabstop>cbName</tabstop>
  <tabstop>pbSelectAll</tabstop>
  <tabstop>pbUnselectAll</tabstop>
  <tabstop>pbDeleteSelected</tabstop>
  <tabstop>pbSearch</tabstop>
  <tabstop>twFileList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbDuplicatesEnbl</sender>
   <signal>clicked(bool)</signal>
   <receiver>gbMethods</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSha1</sender>
   <signal>clicked(bool)</signal>
   <receiver>cbSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSha1</sender>
   <signal>clicked(bool)</signal>
   <receiver>cbSize</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
